apps/chat/src/store/import-export/importExport.epics.ts

```diff
@@ -460,23 +443,25 @@ const uploadConversationAttachmentsEpic: AppEpic = (action$) =>
         return of(ImportExportActions.importFail());
       }
 
-      const importFolderPath = constructPath(getRootId(), ImportRoot.Imports);
+      const conversation = completeHistory.history[firstConversationIndex];
+
+      const importFolderPath = constructPath(
```

Comment:
is it used only for files?
```suggestion
      const importFileFolderPath = constructPath(
```

Reply:
Yes, you are right.
Changed,